### ISSUE TITLE
Как запустить этот шаблон

### ISSUE THREAD

**Issue by RAlex11** [nixel2007/tc-epf-template#8]
Здравствуйте!
Прошу дать более подробную инструкцию по использованию вашего шаблона.
Поставка VA single идет в видео одного epf файла, где то в 1С ее можно подключить как внешнюю обработку. а что потом? куда и как подключать xml файлы, Какая последовательность действий?  Можно ли расписать запуск одного НТ теста подробно по шагам?

**Comment by nixel2007** [comment]
Про саму обработку я рассказывал на докладе: https://infostart.ru/1c/articles/1182048/

**Comment by RAlex11** [comment]
Большое спасибо за ответ! В поисках информации наткнулся на oneScript. То есть собирать проект можно прямо через 1С конфигуратор? или с помощью дополнительного ПО его собрать, а потом добавить как обработку? нужно ли объединять с файлами исходниками VA Single?

**Comment by nixel2007** [comment]
Да, прямо через 1с конфигуратор. Нет, с VA Single объединять не нужно.
